lib/rules/jsx-no-undef-props.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @fileoverview Disallow prop value as `undefined`.
+ * @author Ethan Cohen
+ */
+
+'use strict';
+
+var propName = require('jsx-ast-utils/propName');
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = function(context) {
```

Comment:
This should use the [new eslint rule format](http://eslint.org/blog/2016/07/eslint-new-rule-format)